app/presenters/menu/default_menu.rb

```diff
@@ -23,11 +23,11 @@ def cloud_inteligence_menu_section
           Menu::Item.new('dashboard',  N_('Dashboard'),  'dashboard',  {:feature => 'dashboard_view'},           '/dashboard/'),
           Menu::Item.new('report',     N_('Reports'),    'miq_report', {:feature => 'miq_report', :any => true}, '/report/explorer'),
           # Menu::Item.new('usage',    N_('Usage'),      'usage',      {:feature => 'usage'},                    '/report/usage/'), #  / Hiding usage for now - release 5.2
-          consumption_menu_section,
```

Comment:
Rubocop throw an error here , it seems like the others functions in the file like services_menu_section or clouds_menu_section. Is it necesary fix this?

app/presenters/menu/default_menu.rb

:large_orange_diamond: - Line 35, Col 11 - Style/IndentArray - Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.
:small_blue_diamond: - Line 35, Col 121 - Metrics/LineLength - Line is too long. [135/120]